styles.css

```diff
@@ -1,32 +1,77 @@
 /* ****** GENERAL STYLES ******** */
+
 html {
-  background-color: #A9AAD2;
+  background-color: #EAEAF4;
   font-family: "Open Sans", sans-serif;
   height: 100%;
 }
+
 body {
   display: flex;
```

Comment:
Nice to organize these rules in some way (alphabetically, all positioning stuff grouped, etc.). If there is an agreed-upon convention for this, others can look at the code and easily make edits.  Great job organizing the overall class names and ids, though.